tests/test_api_events_traversal_check.py

```
@@ -0,0 +1,35 @@
+        # Mock Request
+        request = MagicMock(spec=Request)
+
+        # We also need to mock check_db_health to avoid import errors or side effects
+        with patch("swarm.runtime.resilient_db.check_db_health"):
+
+            # Test stream_run_events
+            try:
+                await events.stream_run_events(run_id="../etc", request=request)
+                pytest.fail("Should have raised ValueError for traversal in stream_run_events")
+            except ValueError as e:
+                assert "run_id" in str(e) or "traversal sequence" in str(e)
```

Ruff (0.14.14)

[warning] Found assertion on exception `e` in `except` block, use `pytest.raises()` instead

(PT017)